index.js

```diff
@@ -358,6 +358,8 @@ export default class PercentageCircle extends React.Component<Default, TimerProp
         ]}
       >
         <TextTimeComponent
+          text1={this.props.text1}
+          text2={this.props.text2}
```

Comment:
you might want to consider naming these `pauseText` and `resumeText`.
that way, you know both the functionality (when you will see these text values) as well as the customizability.

Reply:
Sounds good! Just changed it :+1: